docs/src/introduction/overview.md

```diff
@@ -134,139 +166,64 @@ fuel-core --db-type in-memory
 
 This starts a Fuel node with a volatile database that will be cleared when shut down (good for testing purposes).
 
-### Deploy `wallet_contract` To Your Local Fuel Node
+### Deploy `counter_contract` To Your Local Fuel Node
 
-To deploy `wallet_contract` on your local Fuel node, run
+To deploy `counter_contract` on your local Fuel node, open a new terminal tab and run the following command from the root of the `wallet_contract` directory:
 
 ```sh
 forc deploy
 ```
 
-from the root of the `wallet_contract` directory.
+> Note you can't use the same terminal session that is running fuel-core to run any other commands as this will end your fuel-core process.
```

Comment:
```suggestion
> **Note**
> You can't use the same terminal session that is running fuel-core to run any other commands as this will end your fuel-core process.
```